tests/e2e/Fixture.cs

```diff
@@ -38,7 +38,7 @@ public Fixture()
                 .AddJsonFile("appsettings.json")
                 .AddJsonFile($"appsettings.user.json", optional: true)
                 .AddEnvironmentVariables()
-                .AddForgeAlternativeEnvironmentVariables()
+                .AddAPSAlternativeEnvironmentVariables()
```

Comment:
should we allow `AddForgeAlternativeEnvironmentVariables` too?

Reply:
This is just for our E2E test. We don't need either but we had this here for demonstration purposes.